docs/notebooks/verification_gridded_cosmo.ipynb

```diff
@@ -0,0 +1,3105 @@
+    "### 1. Maps"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b61c76fb",
+   "metadata": {},
+   "source": [
+    "**Random Time Selection:** A random time step is selected to avoid bias in the comparison, ensuring that the assessment is representative of typical model performance.\n",
+    "\n",
+    "**Consistent Color Scales:** By setting the same minimum and maximum values across all datasets for each variable, we ensure that color differences in the plots reflect true discrepancies, not artifacts of scaling.\n",
+    "\n",
+    "**Spatial Patterns:** The plots reveal how the ML model and NWP model represent geographical features like weather fronts, high and low-pressure systems, and temperature gradients. Visual comparisons can immediately highlight areas where the models perform well or poorly, guiding further investigation.\n",
+    "\n",
+    "**Edge Effects:** Near the boundaries, artifacts may occur as the model does not calculate a loss in the boundary region."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "76799646",
+   "metadata": {
+    "lines_to_next_cell": 2
+   },
+   "outputs": [],
+   "source": [
+    "ds_boundary = xr.open_zarr(PATH_BOUNDARY)\n",
+    "\n",
+    "ds_boundary = ds_boundary.sel(forcing_feature=list(VARIABLES_BOUNDARY.keys()))\n",
+    "ds_boundary = ds_boundary.sel(split_name=\"test\").drop_dims([\n",
+    "    \"split_part\",\n",
+    "    \"static_feature\",\n",
+    "])\n",
+    "for feature in ds_boundary.forcing_feature.values:\n",
+    "    ds_boundary[VARIABLES_BOUNDARY[feature]] = ds_boundary[\"forcing\"].sel(\n",
+    "        forcing_feature=feature\n",
+    "    )\n",
+    "ds_boundary = ds_boundary.drop_vars([\n",
+    "    \"forcing\",\n",
+    "    \"forcing_feature\",\n",
+    "    \"forcing_feature_units\",\n",
+    "    \"forcing_feature_long_name\",\n",
+    "    \"forcing_feature_source_dataset\",\n",
+    "    \"forcing__train__diff_mean\",\n",
+    "    \"forcing__train__diff_std\",\n",
+    "    \"forcing__train__mean\",\n",
+    "    \"forcing__train__std\",\n",
+    "])\n",
+    "ds_boundary = ds_boundary.set_index(grid_index=[\"latitude\", \"longitude\"])\n",
+    "ds_boundary = ds_boundary.unstack(\"grid_index\")\n",
+    "ds_boundary = ds_boundary.transpose(\"time\", \"latitude\", \"longitude\")\n",
+    "longitude_new = np.where(\n",
+    "    ds_boundary[\"longitude\"] > 180,\n",
+    "    ds_boundary[\"longitude\"] - 360,\n",
+    "    ds_boundary[\"longitude\"],\n",
+    ")\n",
+    "ds_boundary = ds_boundary.assign_coords(longitude=longitude_new).sortby([\n",
+    "    \"longitude\",\n",
+    "    \"latitude\",\n",
+    "])\n",
+    "\n",
+    "lon_mesh, lat_mesh = np.meshgrid(ds_boundary.longitude, ds_boundary.latitude)\n",
+    "ds_boundary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da09d71d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_comparison_maps(\n",
```

Comment:
Should we zoom in a bit on these plots? When the boundary is included it gets really small to actually see the forecast in the interior. Maybe we don,t need to see the full boundary, but just around the region of interest?

Reply:
i really loved to see the convex hull, but my professor also wants to zoom in on the lovely mountains. so I implemented a ZOOM factor the user can define and defaults to 2.